### PR TITLE
feat(atomic,headless): add insight numeric facet

### PR DESCRIPTION
**Pull request by liviarett**
https://user-images.githubusercontent.com/30511433/182808452-9f047ccc-dbb3-4ebf-8cdf-eec36e32aba0.mov

### CODE REVIEW

File: packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx [hunk 1]
```diff
@@ -87,7 +70,7 @@ interface NumericRangeWithLabel extends NumericRangeRequest {
  */
 @Component({
   tag: 'atomic-numeric-facet',
-  styleUrl: 'atomic-numeric-facet.pcss',
+  styleUrl: '../../../common/facets/numeric-facet-common.pcss',
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx:73]: I personally rather keep a pcss file in the same folder & import the common styles from the pcss. This way if there is ever some different styles from 1 facet to the other, we won't need to change this property. The pcss file in the same folder is also the place I think most collaborators would look to first, before checking this property.

What do you think?

File: packages/atomic/src/components/insight/atomic-insight-numeric-facet/atomic-insight-numeric-facet.tsx [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+   * Minimum: `0`
+   * Default: `1000`
+   */
+  @Prop({reflect: true}) public injectionDepth = 1000;
+
+  /**
+   * The required facets and values for this facet to be displayed.
+   * Examples:
+   * ```html
+   * <atomic-facet facet-id="abc" field="objecttype" ...></atomic-facet>
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/insight/atomic-insight-numeric-facet/atomic-insight-numeric-facet.tsx:169]:
```suggestion
   * <atomic-insight-facet facet-id="abc" field="objecttype" ...></atomic-insight-facet>
```

**Review by ThibodeauJF — APPROVED** [review]
Great!

File: packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx (hunk 1 above)
Comment (btaillon) [reply packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx:73]: I agree

**Review by btaillon — APPROVED** [review]

File: packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx (hunk 1 above)
Comment (liviarett) [reply packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx:73]: no worries! will update

**Review by olamothe — APPROVED** [review]
:gg:

**Pull request merged**